scqc_nf.sh

```diff
@@ -108,61 +144,108 @@ process doc {
 
 process mat_merge {
 	publishDir "$params.quantificationsoutdir", mode: 'copy'
+  //cache false
 
 	input:
-	val id_list from finished_ch.collect()
+	// e.g., Finished_LTHSCO2.txt. 
+	path doc_complete_check 
 
 	output:
-	file 'matrix_location.txt' into fortx2g_ch
+	path 'all_plates.tsv'
 	
+	
+	// singularity exec R_verse.v5.img Rscript plate_merge.R quants_dir/ \
+	//	'Finished_CU5DAY0.txt Finished_CU7DAY0.txt Finished_CU5DAY7.txt Finished_CU7DAY7.txt';
 	"""
-	singularity exec ${image_scater} Rscript ${plate_matrix_merge_script} ${params.quantificationsoutdir} \'${id_list}\';
-	echo \$(ls -d -1 ${params.quantificationsoutdir}*.* | grep all) > matrix_location.txt
+	singularity exec ${image_doc} Rscript ${plate_matrix_merge_script} ${params.quantificationsoutdir} ${q_merge_col} \'${doc_complete_check}\';
+	ln -s -v -f ${params.quantificationsoutdir}all_plates.tsv all_plates.tsv
```

Comment:
I have not run the pipeline myself, but here you are using force (`-f`, with `ln -s -v` ) to create a symlink. It will overwrite an existing file. Is that safe in this instance?